File: Java/untitled/src/com/example/FizzBuzz/fizzbuzz.java
package com.example.FizzBuzz;
import java.util.Scanner;

public class fizzbuzz {
    public static void main(String[]args){
        Scanner scan = new Scanner(System.in);
        int bee = scan.nextInt();
        if (bee % 3 == 0 && bee % 5 == 0){
            System.out.println("Fizzbuzz");
        }else if(bee % 3 == 0){
            System.out.print("Fizz");
        } else if (bee % 5 == 0) {
            System.out.println("Buzz");
        }else{
            System.out.println(bee);
        }
    }
}

File: Swift/FizzBuzz/Fizzbuzz.swift
//
//  Fizzbuzz.swift
//  FizzBuzz
//
//  Created by Bryon Carlin on 7/28/22.
//

import Foundation

struct fizzbuzz(bee: Int) -> String {
    switch
    if(bee % 3 == 0, bee % 5 == 0){
    case(true,false):
        return: "Fizz"
    case(false,true):
        return: "Buzz"
    case(true,true):
        return: "FizzBuzz"
    case(false,false):
        return String(bee)
    }
}

print(fizzbuzz(bee: 15))
